packages/vm/src/instance.rs

```diff
@@ -122,6 +122,14 @@ where
             Function::new_native_with_env(store, env.clone(), native_humanize_address),
         );
 
+        // Verifies message hashes against a signature with a public key, using the secp256k1 ECDSA parametrization.
+        // Returns 0 on success. Returns a non-zero memory location to a Region containing an UTF-8 encoded error string for invalid inputs.
+        // Ownership of input pointers is not transferred to the host.
+        env_imports.insert(
+            "verify_secp256k1",
```

Comment:
Let's prefix this with the algorithm name

```suggestion
            "secp256k1_verify",
```